tox.ini

```diff
@@ -19,7 +19,7 @@ commands = flake8 ara
 
 [testenv:py27]
 commands =
-    py.test tests/unit
+    py.test --cov=ara tests/unit
```

Comment:
Could you leave py27 as is and create a tox env called "cover" instead ? This is how OpenStack does it upstream and they have a macro to use it: https://github.com/openstack-infra/project-config/blob/master/jenkins/jobs/python-jobs.yaml#L1